.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.9","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_202.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/udaniel/Library/Application Support/JetBrains/IntelliJIdea2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
I think this is one we need to add to the gitignore and not have here, it has user specific paths in

Reply:
I didn't even notice a new file was generated, good catch. I'll sort it.